x/ecocredit/client/basket/tx.go

```diff
@@ -36,33 +36,28 @@ func TxCreateBasketCmd() *cobra.Command {
 
 Parameters:
-		name: the name used to create a bank denom for this basket token.
+
+- name:  the name used to create a bank denom for this basket token
 
 Flags:
-		disable-auto-retire: disables the auto-retirement of credits upon taking credits
-			from the basket. The credits will be auto-retired if disable_auto_retire is
-			false unless the credits were previously put into the basket by the address
-			picking them from the basket, in which case they will remain tradable.
-		credit-type-abbreviation: filters against credits from this credit type abbreviation (e.g. "BIO").
-		allowed_classes: comma separated (no spaces) list of credit classes allowed to be put in
-			the basket (e.g. "C01,C02").
-		min-start-date: the earliest start date for batches of credits allowed into the basket.
-		start-date-window: the duration of time (in seconds) measured into the past which sets a
-			cutoff for batch start dates when adding new credits to the basket.
-		basket-fee: the fee that the curator will pay to create the basket. It must be >= the
-			required Params.basket_creation_fee. We include the fee explicitly here so that the
-			curator explicitly acknowledges paying this fee and is not surprised to learn that the
-			paid a big fee and didn't know beforehand.
-		description: the description to be used in the basket coin's bank denom metadata.`),
-		Example: `
-		$regen tx ecocredit create-basket HEAED
-			--from regen...
-			--credit-type-abbreviation=FOO
-			--allowed_classes="class1,class2"
-			--basket-fee=100regen
-			--description="any description"
-		`,
-		Args: cobra.ExactArgs(1),
+
+- disable-auto-retire: disables the auto-retirement of credits upon taking credits
+    from the basket. The credits will be auto-retired if disable_auto_retire is
+    false unless the credits were previously put into the basket by the address
+    picking them from the basket, in which case they will remain tradable.
+- credit-type-abbrev: filters against credits from this credit type abbreviation (e.g. "BIO").
+- allowed_classes: comma separated (no spaces) list of credit classes allowed to be put in
+    the basket (e.g. "C01,C02").
+- min-start-date: the earliest start date for batches of credits allowed into the basket.
+- start-date-window: the duration of time (in seconds) measured into the past which sets a
+    cutoff for batch start dates when adding new credits to the basket.
+- basket-fee: the fee that the curator will pay to create the basket. It must be >= the
+    required Params.basket_creation_fee. We include the fee explicitly here so that the
+    curator explicitly acknowledges paying this fee and is not surprised to learn that the
+    paid a big fee and didn't know beforehand.
+- description: the description to be used in the basket coin's bank denom metadata.`),
+		Example: `regen tx ecocredit create-basket NCT --credit-type-abbrev C --allowed_classes C01,C02 basket-fee 100000000uregen description "NCT basket"`,
```

Comment:
I was not entirely sure on the best format for example but ultimately decided no line break before and no line break after so that it is properly formatted in the help text and the generated documentation.

Reply:
Also, this was the only case but no `$` so that it can be easily copied.

Reply:
And this was also the only case, but no multiple line examples (unless multiple examples of course).